Build/15.0/packages.config

```diff
@@ -1,30 +1,52 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.VSSDK.BuildTools" version="15.5.100" />
+  <package id="Microsoft.VSSDK.BuildTools" version="15.7.109" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
   <package id="MicroBuild.Core" version="0.2.0" />
   <package id="Microsoft.VisualStudio.Debugger.InteropA" version="9.0.21022" targetFramework="net46" />
-  <package id="Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime" version="14.3.25408" />
+  <package id="Microsoft.VisualStudio.Debugger.Interop.10.0" version="10.0.30320" />
```

Comment:
Do we need those changes for 15.0?

Reply:
Not strictly, but they're generally good anyway. Right now the 16.0 and 15.0 sets of packages are identical, which is convenient for making it more isolated from the VS install, but I expect them to diverge. Still, we need to keep the builds working with 15.0 until people outside of MS have moved up to 16.0 or else they won't be able to build and use our newest bits against 2017.